app/namespace.go

```diff
@@ -391,6 +394,36 @@ func readAndParseCACerts(ctx *cli.Context) (read caCerts, err error) {
 	return parseCertificates(cert)
 }
 
+func (c *NamespaceClient) failoverNamespace(ctx *cli.Context) error {
+	namespace := ctx.String(NamespaceFlagName)
+	if len(namespace) == 0 {
+		return fmt.Errorf("namespace is required")
+	}
+
+	region := ctx.String(namespaceRegionFlagName)
+	if len(region) == 0 {
+		return fmt.Errorf("region is required")
+	}
+
+	cloudProvider := ctx.String(regionCloudProviderFlagName)
```

Comment:
we should probably just add in 'aws' as default until we're multicloud

Reply:
This is optional